docs/tutorial_volume_annotation.md

```diff
@@ -0,0 +1,56 @@
+# Tutorial: Volume Annotation with WEBKNOSSOS
+
+In this tutorial, we will explore the volume annotation features of WEBKNOSSOS. We will cover everything from opening and annotating a published dataset to utilizing both basic and advanced annotation tools. Additionally, I will explain a few things about meshes and provide insights into the layer system of WEBKNOSSOS. Let’s get started!
+
+<!--
+Here is full tutorial video, alternatively continue reading below.
+
+![type:video Tutorial: Volume Annotation with WEBKNOSSOS](TODO add YouTube link) -->
```

Comment:
Is this still a TODO?

Reply:
Yes, the video has not been release on YouTube et. should come in a week or two